app/main.py

```diff
@@ -2,0 +1,18 @@
+from typing import List
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: List[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The error message you're returning isn't very descriptive. It would be better to provide more information about the error. For example, you could include the name of the friend who isn't vaccinated in the error message. Something like this: `return f'{friend.name} is not vaccinated'`.